src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATE = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from = LocalDate.parse(dateFrom, DATE_FORMATE);
+        LocalDate to = LocalDate.parse(dateTo, DATE_FORMATE);
+        StringBuilder resultStrBuilder = new StringBuilder();
+        resultStrBuilder.append("Report for period ")
+                .append(dateFrom).append(" - ")
+                .append(dateTo)
+                .append(System.lineSeparator());
+
+        for (String name : names) {
```

Comment:
This method is very long, can you please break it down to reduce number of lines, for example cycle could be a separate method.